Controllers/Admin/AdminController.php

```diff
@@ -29,7 +29,6 @@ class AdminController extends Controller
      *
      * @var array
      */
-    protected $helpers = ['form', 'error']; // Load helpers
```

Comment:
Please don't remove the property. Keep the property there like the default BaseController 

protected $helpers = [];